src/modengine/ext/base/base_extension.cpp

```diff
@@ -67,9 +67,9 @@ void ModEngineBaseExtension::on_attach()
     }
 #endif
 
-    const auto dinput8_path = util::system_directory() / "dinput8.dll";
+    const auto dinput8_path = util::system_directory().string() + "dinput8.dll";
```

Comment:
> On the way I had to change a couple things about how system paths are loaded in extensions because I would get a "path too long" error.

Are these changes (i.e. `path / child` -> `string + "child"`) the fix for this, or was this resolved by something else?

Reply:
Yep, that's the ones!